akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -840,7 +845,8 @@ private[akka] class ActorSystemImpl(
       dynamicAccess,
       settings,
       mailboxes,
-      defaultExecutionContext))
+      defaultExecutionContext),
+    log)
 
   val dispatcher: ExecutionContextExecutor = dispatchers.defaultGlobalDispatcher
 
```

Comment:
also search for usages of ActorSystem.dispatcher (e.g. as ExecutionContext for futures) and see if some of those should move over to internal-dipatcher

can also be Typed ActorSystem.executionContext